DSharpPlus.Commands/CommandsExtension.cs

```diff
@@ -215,8 +308,8 @@ private static async Task DefaultCommandErrorHandlerAsync(CommandsExtension exte
         {
             CommandNotFoundException commandNotFoundException => $"Command {Formatter.InlineCode(Formatter.Sanitize(commandNotFoundException.CommandName))} was not found.",
             ArgumentParseException argumentParseException => $"Failed to parse argument {Formatter.InlineCode(Formatter.Sanitize(argumentParseException.Parameter.Name))}.",
-            ChecksFailedException checksFailedException when checksFailedException.FailingContextChecks.Count == 1 => $"Context Check {Formatter.InlineCode(Formatter.Sanitize(checksFailedException.FailingContextChecks.Keys.First().GetType().Name))} failed.",
-            ChecksFailedException checksFailedException => $"The following context checks failed: {string.Join(", ", checksFailedException.FailingContextChecks.Keys.Select(x => Formatter.InlineCode(Formatter.Sanitize(x.GetType().Name))))}.",
+            ChecksFailedException checksFailedException when checksFailedException.Errors.Count == 1 => $"The following error occurred:\n {Formatter.InlineCode(Formatter.Sanitize(checksFailedException.Errors[0]))}",
+            ChecksFailedException checksFailedException => $"The following context checks failed: {Formatter.InlineCode(Formatter.Sanitize(string.Join("\n\n ", checksFailedException.Errors)))}.",
```

Comment:
Extra space one double newline that'll be automatically trimmed by Discord. If this is meant to be a list, add hyphens

Reply:
the list part doesn't work because of the (somewhat questionable) choice to wrap this in `Formatter.InlineCode(Formatter.Sanitize())`, but that's a separate discussion.

Reply:
Ah yeah, you're right. The goal was to wrap just the type/context names with inline codeblocks for readability/accessibility(?).